test/service_consumers/pact_helper.rb

```diff
@@ -0,0 +1,43 @@
+require "pact/provider/rspec"
+require "webmock/rspec"
+require "gds_api/test_helpers/content_store"
+require ::File.expand_path("../../config/environment", __dir__)
+
+Pact.configure do |config|
+  config.reports_dir = "spec/reports/pacts"
+  config.include WebMock::API
+  config.include WebMock::Matchers
+  config.include GdsApi::TestHelpers::ContentStore
+end
+
+class ProxyApp
+  def initialize(real_provider_app)
+    @real_provider_app = real_provider_app
+  end
+
+  def call(env)
+    env["HTTP_HOST"] = "localhost:3003"
+    response = @real_provider_app.call(env)
+    response
+  end
+end
+
+Pact.service_provider "Bank Holidays API" do
+  app { ProxyApp.new(Rails.application) }
```

Comment:
You don't need to use a proxy here. It was needed for the collections pact tests because the results are paginated. See this [commit](https://github.com/alphagov/collections/pull/2230/commits/0a0523fd42ab1787716999938029e23ff7e4bcb7) for details.

So you can lose lines 13 - 23 and line 26

Reply:
You might need to add `"example.org"` to the allowed config.hosts in config/environments/development.rb

Reply:
Yes, we had this discussion [earlier in the PR](https://github.com/alphagov/frontend/pull/2643#discussion_r584565001) and figured that it'd be better to change the Pact tests than to introduce some unexpected configuration. Do you think otherwise?

It's a shame there's not a better way of overriding this in Pact!

Reply:
Yes that makes sense. 

We should probably create a proxy for publishing api then, so we can [lose](https://github.com/alphagov/gds-api-adapters/blob/b5686a840fcc3cac4021b6210272daa89501f1d2/test/publishing_api_test.rb#L2294-L2304) all these stubs

Reply:
I've been starting to think about how we can consolidate and standardise the way we do Pact tests, and I think this is a good example of something we can put in [govuk_test](https://github.com/alphagov/govuk_test). I've got https://trello.com/c/rnfUFP5o/2103-add-common-configuration-for-pact-tests which I'm using to track these sorts of things for now.

Reply:
As a follow-up, we've found we don't need to set the port in the Provider, only in the Consumer: https://github.com/alphagov/frontend/pull/2710